actionpack/lib/action_controller/test_case.rb

```diff
@@ -567,40 +573,68 @@ def paramify_values(hash_or_array_or_value)
       #
       # - +action+: The controller action to call.
       # - +http_method+: Request method used to send the http request. Possible values
```

Comment:
It should be `+method:+`, no ? Also a nit-picky thing but maybe we can use "HTTP" instead of "http".